mlir/tools/rocmlir-tuning-driver/rocmlir-tuning-driver.cpp

```diff
@@ -336,9 +336,26 @@ int main(int argc, char **argv) {
     llvm::errs() << "Could not parse input IR\n";
     return EXIT_FAILURE;
   }
-  if (failed(runTuningLoop(*source))) {
+
+  ModuleOp module;
+  WalkResult findModule = source->walk([&](ModuleOp op) -> WalkResult {
```

Comment:
We've checked to make sure that this correctly handles the root module in non-fusion cases?

Reply:
Yeah, the tuning of Gemm passed.